utils/text_formatting.tsx

```diff
@@ -262,6 +263,16 @@ export function formatText(
         output = `<span class="all-emoji">${output.trim()}</span>`;
     }
 
+    if (postId) {
+        // unwrap the output from an enclosing p tag and add a span that will serve as a
+        // palceholder for `messageToHtmlComponent` function later on
+        if (output.startsWith('<p>') && output.endsWith('</p>')) {
```

Comment:
Do we need to check for the starting p tag or is just checking for the ending one sufficient? I'm trying to think of how a post starting with a list or table would look when turned to HTML.

Reply:
oh, you are absolutely right on this one ... it was due to my first experiments with that, where I somehow came to the conclusion that the whole text is being wrapped in an additional `p` tag. 

Nice catch 👍🏼